apps/logging/management/commands/log_global_state_metrics.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime, timezone
+from django.core.management.base import BaseCommand
+
+from apps.logging.loggers import log_global_state_metrics
+
+class Command(BaseCommand):
+    help = 'Managment command to log global state type metrics when called on a schedule.'
+
+    def handle(self, *args, **options):
+        # Timestamp used to group multiple logging events from this command
+        group_timestamp = datetime.now(timezone.utc).astimezone().replace(microsecond=0).isoformat()
```

Comment:
Added this `group_timestamp` to be used for additional logging events tied to this management command run, even though there is just one currently.  This is so that events can more easily be grouped together for reporting in Splunk.
For example, will be adding events in a future Jira task for per application state information.